### PR TITLE
Improve the description of Conclave codebase structure

### PR DESCRIPTION
**Pull request by anien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,26 +229,26 @@ As work in progress, there are plently of issues and features missing, which we
...
+| [cpp/substratevm/](cpp/substratevm)                                                                                                                                                                                                                | C/C++ enclave code, with the implementation of the entry points (host to enclave) of some EDL code.              |
+| [docs/](docs)                                                                                                                                                                                                                                      | Source code for the [Conclave documentation](https://docs.conclave.net).                                         |
+| [integration&#x2011;tests/](integration-tests)                                                                                                                                                                                                     | A separate independent Gradle project which is used to test the SDK artifacts at an integration-level.           |
```
Comment (JohnJoser3) [inline README.md:248]: Please remove the hyphen in 'integration-level'.

File: README.md [hunk 2]
```diff
@@ -229,26 +229,26 @@ As work in progress, there are plently of issues and features missing, which we
...
+| [plugin&#x2011;enclave&#x2011;gradle/](plugin-enclave-gradle)                                                                                                                                                                                      | The Conclave Gradle enclave plugin which automates the process of building a native SGX binary.                  |
```
Comment (JohnJoser3) [inline README.md:249]: Please add a comma after `The Conclave Gradle enclave plugin`.

File: README.md [hunk 3]
```diff
@@ -229,26 +229,26 @@ As work in progress, there are plently of issues and features missing, which we
 The Conclave Core SDK consists of several technologies working together. Here is a description of the most
 meaningful directories in this repo.
```
Comment (JohnJoser3) [inline README.md:230]: The word _meaningful_ doesn't seem to fit here. Please rewrite as: `Here is a description of the most _important_ directories in this repo`.

**Review by JohnJoser3 — CHANGES_REQUESTED** [review]
Nice changes! A few minor suggestions from my side.

File: README.md (hunk 2 above)
Comment (anien) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (anien) [reply README.md]: Done.

**Comment by anien** [comment]
PTAL.

**Review by JohnJoser3 — APPROVED** [review]
Looks good to me.

**Pull request merged**